src/megatron/bridge/models/conversion/utils.py

```diff
@@ -277,7 +277,10 @@ def get_causal_lm_class_via_auto_map(
                 repo_id=repo_id,
             )
         except Exception:
-            return None
+            # If dynamic loading fails (e.g., version incompatibilities),
+            # return the class name as string for string-based bridge registration
+            class_name = auto_map_class.split(".")[-1]  # Extract class name from module.ClassName
+            return class_name
```

Comment:
@yaoyu-33 @yfw I needed this fallback to register "DeciLMForCausalLM", which is at odds with https://github.com/NVIDIA-NeMo/Megatron-Bridge/pull/937